init.js

```diff
@@ -0,0 +1,71 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const assert = require('assert');
+const fs = require('fs');
+const os = require('os');
+
+const async = require('async');
+const ioctl = require('ioctl');
+const constants = require('./constants').default;
+const config = require('./lib/Config.js').default;
+const logger = require('./lib/utilities/logger.js').logger;
+
+function _setDirSyncFlag(path) {
+    const GETFLAGS = 2148034049;
+    const SETFLAGS = 1074292226;
+    const FS_DIRSYNC_FL = 65536;
+    const buffer = new Buffer(8);
+    const pathFD = fs.openSync(path, 'r');
+    const status = ioctl(pathFD, GETFLAGS, buffer);
+    assert.strictEqual(status, 0);
+    const currentFlags = buffer.readUIntLE(0, 8);
+    const flags = currentFlags | FS_DIRSYNC_FL;
+    buffer.writeUIntLE(flags, 0, 8);
+    const status2 = ioctl(pathFD, SETFLAGS, buffer);
+    assert.strictEqual(status2, 0);
+    const confirmBuffer = new Buffer(8);
+    ioctl(pathFD, GETFLAGS, confirmBuffer);
+    assert.strictEqual(confirmBuffer.readUIntLE(0, 8),
```

Comment:
Not a big deal, but we could arguably want to do this assert on the value retrieved from a closed and re-opened fd.